pyupgrade/_plugins/unittest_aliases.py

```diff
@@ -56,10 +57,13 @@ def visit_Call(
         )
         yield ast_to_offset(node.func), func
     elif (
-            isinstance(node.func, ast.Attribute) and
-            isinstance(node.func.value, ast.Name) and
-            node.func.value.id == 'unittest' and
-            node.func.attr in FUNCTION_MAPPING
```

Comment:
don't touch unrelated code to imbue your style on it

Reply:
oops, left formatter settings on, sorry